src/models/addw1/README.md

```diff
@@ -1,26 +0,0 @@
-# Adder WS (addw1)
```

Comment:
Was deleting these files intentional?

Reply:
No.

Reply:
yes, they shouldn't have been this branch. They were causing conflicts with the master branch.